docs/data-analysis.md

```diff
@@ -33,55 +99,78 @@ df.groupby("pair")["sell_reason"].value_counts()
 
 ### Load multiple configuration files
 
-This option can be usefull to inspect the results of passing in multiple configs in case of problems
+This option can be useful to inspect the results of passing in multiple configs
 
 ``` python
+# Load config from multiple files
 from freqtrade.configuration import Configuration
 config = Configuration.from_files(["config1.json", "config2.json"])
-print(config)
-```
 
-## Strategy debugging example
+# Show the config in memory
+import json
+print(json.dumps(config, indent=1))
+```
 
-Debugging a strategy can be time-consuming. FreqTrade offers helper functions to visualize raw data.
+### Load exchange data to a pandas dataframe
 
-### Import requirements and define variables used in analyses
+This loads candle data to a dataframe
 
 ```python
-# Imports
+# Load data using values passed to function
 from pathlib import Path
-import os
 from freqtrade.data.history import load_pair_history
-from freqtrade.resolvers import StrategyResolver
 
-# You can override strategy settings as demonstrated below.
+ticker_interval = "5m"
+data_location = Path('user_data', 'data', 'bitrex')
+pair = "BTC_USDT"
+candles = load_pair_history(datadir=data_location,
+                            ticker_interval=ticker_interval,
+                            pair=pair)
+
+# Confirm success
+print("Loaded " + str(len(candles)) + f" rows of data for {pair} from {data_location}")
+display(candles.head())
+```
+
+## Strategy debugging example  
+
+Debugging a strategy can be time-consuming. FreqTrade offers helper functions to visualize raw data.
+
+### Define variables used in analyses  
+
+You can override strategy settings as demonstrated below.
+
+```python
 # Customize these according to your needs.
 
 # Define some constants
 ticker_interval = "5m"
 # Name of the strategy class
-strategy_name = 'AwesomeStrategy'
+strategy_name = 'TestStrategy'
 # Path to user data
 user_data_dir = 'user_data'
 # Location of the strategy
 strategy_location = Path(user_data_dir, 'strategies')
 # Location of the data
 data_location = Path(user_data_dir, 'data', 'binance')
-# Pair to analyze 
-# Only use one pair here
+# Pair to analyze - Only use one pair here
 pair = "BTC_USDT"
 ```
 
 ### Load exchange data
 
 ```python
 # Load data using values set above
-bt_data = load_pair_history(datadir=Path(data_location),
+from pathlib import Path
+from freqtrade.data.history import load_pair_history
+
+candles = load_pair_history(datadir=data_location,
                             ticker_interval=ticker_interval,
                             pair=pair)
 
 # Confirm success
-print(f"Loaded {len(bt_data)} rows of data for {pair} from {data_location}")
+print("Loaded " + str(len(candles)) + f" rows of data for {pair} from {data_location}")
+display(candles.head())
```

Comment:
What's the advantage of using `display()` as oposed to using `candles.head()` directly?

it limits us to jupyter notebooks, while the scripts would otherwise work in a REPL as well.

Reply:
It's a habit. In this case, I can remove `display()` but I typically call `display()` explicitly for the following reasons:
* one can set an option to turn off the printing of output that is not wrapped in `print()` or `display()` If that is set, a user may not realize that output is supposed to show.
* Jupyter notebooks only print the last implied print statement in a cell. So, 

```python
# This only prints the tail
df.head()
df.tail()

# this prints the head and tail
display(df.head())
display(df.tail())
```
* `display()` pretty-prints the object with the Jupyter styling system whereas implied print statements like `df.head()` or print statements like `print(df.head())` produce console output that often mess up line endings, embedded hrefs and other elements that are formatted by Jupyter styling.

Reply:
I wasn't aware of the following usecase - incorporated that to my workflow now ... 

``` python
display(df.head())
display(df.tail())
```

maybe leave `display()` in the notebook, but remove it here (even though that will create a disconnect ...)?

Reply:
Sure. It this or the f-strings show up again, let me know :).